code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -13,17 +13,22 @@
 	attack_verb = list("bashed", "battered", "bludgeoned", "thrashed", "whacked")
 	burn_state = FLAMMABLE
 	burntime = 20
+	var/folder_path = /obj/item/weapon/folder //this is the path of the folder that gets spawned in New()
 
 /obj/item/weapon/storage/briefcase/New()
 	..()
-	new /obj/item/weapon/paper(src)
-	new /obj/item/weapon/paper(src)
-	new /obj/item/weapon/paper(src)
-	new /obj/item/weapon/paper(src)
-	new /obj/item/weapon/paper(src)
-	new /obj/item/weapon/paper(src)
 	new /obj/item/weapon/pen(src)
+	var/obj/item/weapon/folder/folder = new folder_path(src)
+	for(var/i in 1 to 6)
+		new /obj/item/weapon/paper(folder)
 
+/obj/item/weapon/storage/briefcase/lawyer
+	folder_path = /obj/item/weapon/folder/blue
+
+/obj/item/weapon/storage/briefcase/lawyer/New()
+	new /obj/item/weapon/stamp/law(src)
+	..()

```

Comment:
This way, it will first get the law stamp, then the pen and the folder. The other way it would get the pen, the folder, then the law stamp. Both ways would work, if I'm not mistaken.